src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_rsa.c

```diff
@@ -217,21 +228,7 @@ PALEXPORT int32_t CryptoNative_RsaVerificationPrimitive(int32_t flen, uint8_t* f
     return (int32_t)decryptedBytesLen;
 }
 
-PALEXPORT int32_t CryptoNative_RsaSign(int32_t type, uint8_t* m, int32_t mlen, uint8_t* sigret, int32_t* siglen, RSA* rsa)
-{
-    // TODO:
-    LOG_ERROR("RSA signing NYI");
-    return RSA_FAIL;
-}
-
-PALEXPORT int32_t CryptoNative_RsaVerify(int32_t type, uint8_t* m, int32_t mlen, uint8_t* sigbuf, int32_t siglen, RSA* rsa)
-{
-    // TODO:
-    LOG_ERROR("RSA signing NYI");
-    return RSA_FAIL;
-}
-
-PALEXPORT int32_t CryptoNative_RsaGenerateKeyEx(RSA* rsa, int32_t bits, jobject pubExp)
+PALEXPORT int32_t AndroidCryptoNative_RsaGenerateKeyEx(RSA* rsa, int32_t bits, jobject pubExp)
```

Comment:
`pubexp` looks like it's held by this method, but not actually used in the key generation process.  So we should get rid of it.  (It was only here because OpenSSL's RSA key generator required it as a formal parameter)